### ISSUE TITLE
ConfigFile unit test failing locally, works on AppVeyor

### ISSUE THREAD

**Issue by drewnoakes** [gitextensions/gitextensions#5233]
**Do you want to request a *feature* or report a *bug*?**

🐛 

**What is the current behavior?**

`ConfigFileTest.CaseSensitive` fails on my machine, but passes on AppVeyor. It's been this way for a while (a month perhaps). I've taken a brief look and haven't determined the cause of the problem. Does anyone else see this?

```text
  String lengths are both 7. Strings differ at index 6.
  Expected: "origin2"
  But was:  "origin3"
  -----------------^

   at GitCommandsTests.Config.ConfigFileTest.CaseSensitive() in ConfigFileTest.cs:line 750
```

**Environment you encounter the issue:**

- GitExtensions version: master (c048fe741ddb0d079074f28b228815ef22d02592)
- GIT version: 2.18
- OS version: Windows 10
- .NET version: 4

**Did this work in previous version of GitExtensions (which)?**

Yes. Not sure when this started happening exactly.

**Comment by RussKie** [comment]
![image](https://user-images.githubusercontent.com/4403806/43315588-c3c987ae-91d9-11e8-82a5-e1474b57d37c.png)

at a27ebc97be568917880db958c940a6a244ba580d

**Comment by gerhardol** [comment]
Fails for me too, as well as at least two other tests

**Comment by jbialobr** [comment]
Fixed in #5389
